codelighthouse/CodeLighthouse.py

```diff
@@ -68,20 +68,23 @@ def error(self, exception, email=None, args=None, kwargs=None):
         :return: The error guid if it was successful -- otherwise, returns None
         """
         arguments = CodeLighthouse.format_arguments(args, kwargs)
-        stack_trace = CodeLighthouse.format_stack_trace(exception.__traceback__)
+        parsed_stack_trace = CodeLighthouse.format_stack_trace(exception.__traceback__)
+        stack_trace = ''.join(traceback.format_exception(None, exception, exception.__traceback__))
+        print(stack_trace)
```

Comment:
I don't think we should be printing the stack trace here, should we?

Reply:
lol yeah that'd be my fault